mandatory/2-function-creation.js

```diff
@@ -5,7 +5,12 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  for (let i = 0; i < strArr.length; i++) {
+    strArr[i] = strArr[i].trim().replace("/", "").toLowerCase();
```

Comment:
Well done on doing the trim, replacement, and coversion to lower case, all in one line!